lib/puppet/parser/ast/resource_instance.rb

```diff
@@ -4,7 +4,7 @@
 class Puppet::Parser::AST::ResourceInstance < Puppet::Parser::AST::Branch
   attr_accessor :title, :parameters
   def initialize(argshash)
-    Puppet.warn_once('deprecation', 'AST::ResourceInstance', 'Use of Puppet::Parser::AST::ResourceInstance is deprecated')
+    Puppet.warn_once('deprecation', 'AST::ResourceInstance', _('Use of Puppet::Parser::AST::ResourceInstance is deprecated'))
```

Comment:
`Puppet::Parser::AST::ResourceInstance` contains several function/classname keywords, should we comment about not translating it?